docs: make a note about size/type change relationship in Redshift (#365)
snowplow/documentation: docs/understanding-tracking-design/versioning-your-data-structures/enterprise/index.md

:::caution

> In Redshift, changing _size_ may also mean _type_ change; e.g. changing the `maximum` integer from say `30000` to `100000`. See our documentation on [how schemas translate to database types](/docs/understanding-tracking-design/json-schema-type-casting-rules/index.md), as well as Redshift’s documentation on [numeric types](https://docs.aws.amazon.com/redshift/latest/dg/r_Numeric_types201.html).

Reviewer: Actually, this is true not only for Redshift, but also for Databricks.

> In Redshift, changing _size_ may also mean _type_ change; e.g. changing the `maximum` integer from say `30000` to `100000`. See our documentation on [how schemas translate to database types](/docs/understanding-tracking-design/json-schema-type-casting-rules/index.md), as well as Redshift’s documentation on [numeric types](https://docs.aws.amazon.com/redshift/latest/dg/r_Numeric_types201.html).

Reviewer: ```suggestion
In Redshift and Databricks, changing _size_ may also mean _type_ change; e.g. changing the `maximum` integer from `30000` to `100000`. See our documentation on [how schemas translate to database types](/docs/understanding-tracking-design/json-schema-type-casting-rules/index.md).
```

> * **NOTE:** In Redshift, changing _size_ may also mean _type_ change; e.g. changing the `maximum` integer from say `30000` to `100000` (see [Numeric types - Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/dg/r_Numeric_types201.html))

Reviewer 2: ```suggestion
:::caution

In Redshift, changing _size_ may also mean _type_ change; e.g. changing the `maximum` integer from say `30000` to `100000` (see [Numeric types - Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/dg/r_Numeric_types201.html))

:::
